src/frontend/src/routes/(split)/satellite/+page.svelte

```diff
@@ -42,15 +42,15 @@
 </script>
 
 <IdentityGuard>
-	<Tabs>
-		{#snippet info()}
-			{#if nonNullish($satelliteStore)}
-				<Warnings satellite={$satelliteStore} />
-			{/if}
-		{/snippet}
+	<SatelliteGuard>
+		<Tabs>
```

Comment:
This is still incorrect. The `<Loaders />` should not move.

Other components are ok.